src/Calculator.js

```diff
@@ -0,0 +1,125 @@
+import './App.css';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Button from 'react-bootstrap/Button';
+
+function Calculator() {
+  return (
+    <Container className="d-grid gap-2 mt-5 pt-5">
+      <Row>
```

Comment:
- [ ] You've done an excellent job, keep it up! Let's refactor this function to make it a class-based function as this is a requirement for this activity:

![Screen Shot 2022-09-15 at 11 48 57 AM](https://user-images.githubusercontent.com/83717285/190461759-e79bf3fc-797c-433d-9973-63af34b08820.png)